UnityProject/Assets/Scripts/Messages/Client/mapping/ClientRequestLoadMap.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using MapSaver;
+using Messages.Client;
+using Mirror;
+using Newtonsoft.Json;
+using TileManagement;
+using UnityEngine;
+
+public class ClientRequestLoadMap : ClientMessage<ClientRequestLoadMap.NetMessage>
+{
+	public static Dictionary<PlayerInfo, List<string>> SaveDatas = new Dictionary<PlayerInfo, List<string>>();
+
+	public struct NetMessage : NetworkMessage
+	{
+		public Vector3 Offset00;
+		public Vector3 Offset;
+		public string Data;
+		public bool end;
```

Comment:
end to what?

Reply:
because it gets Chunked because it's too big for one message